### PR TITLE
Rally tag fix: truncate tags to 32 characters

### PR DESCRIPTION
**Pull request by geoffwhittington**

### CODE REVIEW

**Comment by geoffwhittington** [comment]
@ehsanf Fix is in

File: sdetools/modules/sync_rally/rally_plugin.py [hunk 1]
```diff
@@ -332,6 +335,10 @@ def rally_get_artifact(self, name, query, card_type, artifact_type, api):
             raise AlmException('Unable to get artifact from Rally. Reason: %s' % (str(err)))
         return task_data[artifact_type]
 
+    def _transform_label_to_rally_tag(self, label):
+        """ Rally supports tags having at most 32 characters """
+        return label[0:32]
```
Comment (funkaoshi) [inline sdetools/modules/sync_rally/rally_plugin.py]: `return label[:32]` is probably more pythonic? People usually drop the starting zero if you are slicing from the start of the list.

File: sdetools/modules/sync_rally/rally_plugin.py [hunk 2]
```diff
@@ -213,6 +213,9 @@ def alm_connect_project(self):
 
     def _setup_rally_label(self, tag_name):
 
+        # Ensure tags are transformed
```
Comment (funkaoshi) [inline sdetools/modules/sync_rally/rally_plugin.py]: ` ... because Rally only supports 32 character tags`

Should say why we need to do this.

**Comment by funkaoshi** [comment]
+1

**Pull request merged**